### PR TITLE
Add web version of VS Code

### PR DESCRIPTION
**Pull request by ForumPlayer**

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -402,6 +402,7 @@ gifyourgame.com
 giggl.app
 gikken.co
 giphy.com
+github.dev
```
Comment (Gusted) [inline src/config/dark-sites.config:405]: Not dark-by-default.

image

Comment (ForumPlayer) [reply src/config/dark-sites.config:405]: Maybe it respects what you selected on the main GitHub domain, did you selected the GitHub to use the dark theme?

**Comment by ForumPlayer** [comment]
By now i'll remove the GitHub Codespaces domain from the commit, since the VS Code is on the dark theme by default

File: src/config/dark-sites.config (hunk 1 above)
Comment (Gusted) [reply src/config/dark-sites.config:405]: Nope light-mode. But that isn't dark-by-default.

Comment (ForumPlayer) [reply src/config/dark-sites.config]: Honestly, I think if you're going to open the a Codespace, you already have a GitHub account and (probably) put the main website on the Dark mode too.
But, that's my personal opinion, and I can't speak for everyone.

Anyways, I already pushed the new commit fixing it

**Review by Gusted — APPROVED** [review]

**Pull request merged**